AGENTS.md

```
@@ -202,16 +157,13 @@ bun audit
 ```
 
 ### 3.4 Rollback Procedures
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)